client/components/date-range/style.scss

```diff
@@ -321,7 +321,7 @@ $date-range-mobile-layout-switch: $break-small;
 	}
 
 	@media (max-width: $date-range-mobile-layout-switch) {
-		flex-direction: column; // Stack items vertically on small screens
+		flex-direction: column-reverse; // Stack items vertically on small screens
```

Comment:
@annacmc it seems to work well by reverse the column 😄

Reply:
Oh nice one! That works really well! :)